### PR TITLE
Added Admin pages

### PR DESCRIPTION
**Pull request by pragya-joshi**

### CODE REVIEW

File: client/src/pages/AddEmployee.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+            >
+              Back to Dashboard
+            </button>
+          </Link>
+          <div className="row d-flex justify-content-center align-items-center h-100">
+            <div className="col-lg-8 col-xl-6">
+              <div className="card rounded-3">
+                <div className="card-body p-4 p-md-5">
+                  <h3 className="mb-4 pb-2 pb-md-0 mb-md-5 px-md-2">
+                    Add a new Employee
```
Comment (ankitmishra03) [inline client/src/pages/AddEmployee.js:63]: add a const function for each div, and use that component here, for more readability and cleanliness  of code.

File: client/src/pages/AddJob.js [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+              style={{ marginTop: "20px", marginRight: "20px" }}
+            >
+              Back to Dashboard
+            </button>
+          </Link>
+          <div className="row d-flex justify-content-center align-items-center h-100">
+            <div className="col-lg-8 col-xl-6">
+              <div className="card rounded-3">
+                <div className="card-body p-4 p-md-5">
+                  <h3 className="mb-4 pb-2 pb-md-0 mb-md-5 px-md-2">
```
Comment (ankitmishra03) [inline client/src/pages/AddJob.js:60]: add const functions and use that here for every div.

File: server/controllers/employees.js [hunk 3]
```diff
@@ -21,18 +21,22 @@ const getEmployeeFromId = async (req, res) => {
 const createNewEmployee = async (req, res) => {
   try {
     const employee = await Employees.create(req.body);
-    res.status(201).json(employee);
+    res.status(201).json({ employee });
```
Comment (ankitmishra03) [inline server/controllers/employees.js:24]: why?

**Review by ankitmishra03 — CHANGES_REQUESTED** [review]

File: server/controllers/employees.js (hunk 3 above)
Comment (pragya-joshi) [reply server/controllers/employees.js]: it should be only employee.....{employee} was basically creating a json object with a key employee. Changed the code to employee

**Review by bhavneet2000 — APPROVED** [review]

**Pull request merged**